src/test/java/ua/com/juja/microservices/keepers/slackbot/model/SlackParsedCommandTest.java

```diff
@@ -18,24 +17,26 @@
  * @author Konstantin Sergey
  */
 public class SlackParsedCommandTest {
-    private Map<String, UserDTO> users;
+    private List<UserDTO> usersInText;
+    private UserDTO fromUser;
 
     @Rule
     public ExpectedException thrown = ExpectedException.none();
 
     @Before
     public void setup() {
-        users = new HashMap<>();
-        users.put("@from", new UserDTO("uuid0", "@from"));
-        users.put("@slack1", new UserDTO("uuid1", "@slack1"));
-        users.put("@slack2", new UserDTO("uuid2", "@slack2"));
+        fromUser = new UserDTO("uuid0", "@from");
+        usersInText = new ArrayList<>();
+        usersInText.add(new UserDTO("uuid1", "@slack1"));
+        usersInText.add(new UserDTO("uuid2", "@slack2"));
+        usersInText.add(new UserDTO("uuid3", "@slack3"));
```

Comment:
I think the list for each test should be contains specific users accordance with the text.
for example 
"text text @slack1 text" 
usersInText must contains only one object UserDTO